HaulageApplication/Haulage/BaseClasses/TripHandler/Item.cs

```diff
@@ -6,13 +6,32 @@
 
 namespace Haulage.BaseClasses.TripHandler
 {
+    
     public class Item
     {
+        public enum ItemCategoryType
+        {
+            Fragile,
+            Dangerous,
+            None
+        }
+        //public enum RequestStatusType
+        //{
+        //    pickUpRequested,
+        //    EnRouteToPickup,
+        //    PickedUp,
+        //}
         public Guid itemID;
         public string description;
-        public bool itemCategory;
+        public ItemCategoryType itemCategory;
         public bool signedOff;
         public Item() { }
+        public int ItemId { get; set; }
+        public string? Name { get; set; }
+        public string? Description { get; set; }
+        public bool SignedOff { get; set; }
+        public ItemCategoryType ItemCategory { get; set; }
+        public PickupRequest.requestStatus RequestStatus { get; set; }
```

Comment:
not your change but if RequestStatus is being used in other classes except pickupRequest it should probably be moved out of class